### PR TITLE
Update examples to remove deprecated fields

### PR DESCRIPTION
**Pull request by bookernath**
## What changed?
* Remove mention of deprecated fields from examples
* Minor clarifications
* Update examples to reduce the total number and remove deprecated fields
* Clean up code so you can actually read the queries

### CODE REVIEW

**Review by aglensmith — APPROVED** [review]

**Review by aglensmith — DISMISSED** [review]

**Review by zvuki — COMMENTED** [review]
I know it's tempting to showcase all the great features of graphql and to demonstrate how much you can do with a single request. However, as many of these queries will be likely copy-pasted, I'd encourage to make them as simple as possible, so they can be adjusted to an actual use case rather than just used blindly as is. It's hard to say for sure how much impact sample queries have, but it should help the platform to avoid unnecessary load to a certain extend.

File: assets/js/playground_tabs.js [hunk 1]
```diff
@@ -9,76 +9,493 @@
...
+          name
+          description
+          images {
+            edges {
+              node {
+                url(width: 500, height: 500)
+              }
+            }
+          }
+          brand {
```
Comment (zvuki) [inline assets/js/playground_tabs.js:93]: can be removed (?)

File: assets/js/playground_tabs.js [hunk 2]
```diff
@@ -9,76 +9,493 @@
...
+          }
+          ... on BulkPricingPercentageDiscount {
+            percentOff
+          }
+          ... on BulkPricingRelativePriceDiscount {
+            priceAdjustment
+          }
+        }
+      }
+      brand {
```
Comment (zvuki) [inline assets/js/playground_tabs.js:163]: can be removed (?)

File: assets/js/playground_tabs.js [hunk 3]
```diff
@@ -9,76 +9,493 @@
...
+}`
+
+var extendedProductQuery =
+`# Get extended information (options, variants) for several products with a list of IDs
+query ExtendedProductsById(
+  $productIds: [Int!] 
+  # Use GraphQL Query Variables to inject your product IDs
+) {
+  site {
+    products(entityIds: $productIds, first:25) {
```
Comment (zvuki) [inline assets/js/playground_tabs.js:237]:
```suggestion
    products(entityIds: $productIds, first:5) {

```

File: assets/js/playground_tabs.js [hunk 4]
```diff
@@ -9,76 +9,493 @@
...
+      edges {
+        node {
+          name
+          variants(first: 25) {
```
Comment (zvuki) [inline assets/js/playground_tabs.js:241]:
```suggestion
          variants(first: 5) {

```

File: assets/js/playground_tabs.js [hunk 5]
```diff
@@ -9,76 +9,493 @@
...
+            edges {
+              node {
+                sku
+                defaultImage {
+                  url(width: 1000)
+                }
+                productOptions(first: 5) {
```
Comment (zvuki) [inline assets/js/playground_tabs.js:248]: can be removed (?)

File: assets/js/playground_tabs.js [hunk 6]
```diff
@@ -9,76 +9,493 @@
...
+                      key
+                      id
+                      value
+                    }
+                  }
+                }
+              }
+            }
+          }
+          brand {
```
Comment (zvuki) [inline assets/js/playground_tabs.js:391]: can be removed (?)
demonstrating products/variants might be sufficient

File: assets/js/playground_tabs.js (hunk 2 above)
Comment (bookernath) [reply assets/js/playground_tabs.js:163]: I think it's relevant here - trying to show a more complete product object. for PDP use cases.

File: assets/js/playground_tabs.js (hunk 1 above)
Comment (bookernath) [reply assets/js/playground_tabs.js:93]: done

**Review by uwikaiddi — APPROVED** [review]

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**